src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -0,0 +1,1127 @@
+[
+  {
+    "name": "graphql.language.OperationDefinition",
+    "queryAllDeclaredConstructors": true,
+    "queryAllPublicMethods": true,
+    "allPublicFields": true,
+    "allDeclaredMethods": true,
+    "allPublicMethods": true,
+    "allPublicConstructors": true,
+    "queryAllDeclaredMethods": true,
+    "queryAllPublicConstructors": true,
+    "allDeclaredClasses": true,
+    "allDeclaredFields": true,
+    "allPublicClasses": true,
+    "allDeclaredConstructors": true
+  },
```

Comment:
- It looks like this is not generated for unit tests via the GraalVM Native Build Tools' GraalVM Agent, but handled manually?  This looks different from `oracle/graalvm-reachability-metadata`. 👀

Reply:
Anybody can author the Config files; you don't need to use the java agent. I'm not sure what you mean by 'for unit tests'. These config files work in any context: I'm most interested in production code. I arrived at them by debugging a Spring for GraphQL application (and trying to make the GraphiQL console work). These Config files mean we can introspect GraphQL using GraphQL

Reply:
- No problem, I just assumed commands like `./gradlew -Pagent run` and `./gradlew metadataCopy --task run --dir src/main/resources/META-INF/native-image` were used.  I understand that the metadata generated by `Conditional Agent Mode` is usually shorter, refer to https://graalvm.github.io/native-build-tools/latest/gradle-plugin.html#agent-support . 😃